R/plot_fns.R

```diff
@@ -3604,18 +3710,24 @@ plot.rmdFilt <- function (filter_object, pvalue_threshold = NULL, sampleID = NUL
 
   # determine the main effects, then melt the df #
   if (ncol(group_df) == 2) {
-
     main_eff_names <- "Group"
-    dfmelt <- reshape2::melt(filter_object, id = c(samp_id, main_eff_names))
-
+    dfmelt <- filter_object %>%
+      tidyr::pivot_longer(
+        -dplyr::all_of(c(!!samp_id, !!main_eff_names)),
```

Comment:
Note to check: can any other !!sym() calls be replaced with this notation?

Reply:
Perhaps, we can track them down later if needed.  I think I use it a lot in filter(), not sure I need to be.